README.md

```diff
@@ -45,6 +45,24 @@ $ cekit build --overrides "{version: <version_default_is_latest>}" docker
 
 ### Running the Infinispan Operator
 
+#### Adding Annotations
+
+Annotations are labels that {ispn_operator} propagates to pods and services that it controls.
+You should define custom annotations before you install the Infinispan Operator with the following environment variables:
+
+```
+INFINISPAN_OPERATOR_TARGET_LABELS - Defines annotations for services.
```

Comment:
This is just set of the target label and it's value propagated on the operator level globally for the services